docs/protocol.md

```diff
@@ -322,21 +322,36 @@ POST / HTTP/1.1
 #### Original DID Document example
 ```json
 {
+  "id": "someIdasdf",
   "@context": "https://w3id.org/did/v1",
```

Comment:
This fix the outdated documentation. We now require some fields that were not required.